templates/boot-fastboot/generic-fastboot-boot-template.jinja2

```diff
@@ -0,0 +1,85 @@
+{%- extends 'base/kernel-ci-base.jinja2' %}
+{%- block protocols %}
+protocols:
+  lava-lxc:
+    name: lxc-{{ platform }}-art-conv
+    template: debian
+    distribution: debian
+    release: jessie
+    mirror: http://mirror.bytemark.co.uk/debian
```

Comment:
Please remove the mirrors.  The UK mirror is not relevant for all labs.

Reply:
oky

Reply:
See also: http://deb.debian.org/

Reply:
we can remove the mirror because we are not using a real lxc it's mocked.